src/app/features/public-form/edit-publicform-route/edit-publicform-route.component.html

```diff
@@ -0,0 +1,32 @@
+<form [formGroup]="form">
+  <div class="flex-row">
+    <mat-form-field
+      appearance="outline"
+      [class.disabled]="formControl.disabled"
+      floatLabel="always"
+    >
+      <mat-label i18n>Form Link ID</mat-label>
```

Comment:
one thought about UX complexity:

We could show the tooltip explanation which is currently a big "?" button on the right as a small tooltip with the label here. Like in this example:
![image](https://github.com/user-attachments/assets/bb00cba0-0818-4b42-8c77-e4f468c76488)

Then the component would have only one big icon on the right and is therefore maybe less overwhelming?

What do you think, @george-neha , @Abhinegi2 ?

Reply:
I think moving the tooltip explanation makes the interface cleaner.